spring-cloud-dataflow-admin-local/src/test/java/org/springframework/cloud/dataflow/admin/spi/local/AdminDefaultTestEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.admin.spi.local;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+public class AdminDefaultTestEnvironmentPostProcessor implements EnvironmentPostProcessor {
```

Comment:
What is the rationale behind this? It seems like a dangerous idea to make a significant change to how the properties work within tests since that means we are not testing the same behavior that would occur in real world usage.